docs/source/supported/anvil.rst

```diff
@@ -0,0 +1,117 @@
+================
+Anvil (Purdue)
+================
+
+Platform user guide
+===================
+
+https://www.rcac.purdue.edu/knowledge/anvil
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``purdue.anvil`` - ``SRUN``
+
+* Configuration per node
```

Comment:
it actually got a little confusing, it's better to present it different way - similar as it is done for Delta and Perlmutter.

There are 3 configurations (1000 CPU-only nodes, 32 CPU-only nodes with large memory, and 16 GPU-nodes) and for each to have a separate platform, at least for 2 configurations, e.g., `purdue.anvil`, `purdue.anvil_gpu`

Reply:
Since we are doing the seprarion based on how the machine group its nodes, I suggest to follow Anvil categorization and do the following: `purdue.anvil`, `purdue.anvil_gpu`, `purdue.anvil_mem` or `purdue.anvil_highmem`.